src/main/java/org/jboss/pnc/common/Numbers.java

```diff
@@ -46,6 +49,13 @@ public static String decimalToBase64(long l) {
         return encoder.encodeToString(longToBytes(l));
     }
 
+    /**
+     * Convert long to base32
+     */
+    public static String decimalToBase32(long l) {
+        return base32.encodeToString(longToBytes(l)).replace("=", "");
```

Comment:
what about generating base32 string without the padding `base32 = new Base32((byte)0)`

Reply:
I was thinking I would try that but I was afraid it would create string with `\0\0` at the end.

Reply:
yup, it is returning `\0`s:
![image](https://user-images.githubusercontent.com/644267/118111489-0e5fb200-b3e4-11eb-90e8-8dc05392b671.png)